### PR TITLE
[Bexley] Update Symology status mapping

### PR DESCRIPTION
**Pull request by davea**
For FD-3182

### CODE REVIEW

File: perllib/Open311/Endpoint/Integration/UK/Bexley/Symology.pm [hunk 1]
```diff
@@ -72,7 +72,7 @@ sub _update_status {
             'fixed'
         } elsif ($action_due =~ /^[NS][1-6]$/) {
             'in_progress'
-        } elsif ($action_due =~ /^IR|REH|RES|RET|RP|RPOS|RT|RWT$/) {
+        } elsif ($action_due =~ /^IR|RBC|REH|RES|RET|RP|RPL|RPOS|RSW|RT|RWT$/) {
```
Comment (dracos) [inline perllib/Open311/Endpoint/Integration/UK/Bexley/Symology.pm:75]: Not sure if this is deliberate, but the lack of brackets (before) means this matched anything starting IR, anything ending RWT, plus anything containing the rest (so e.g. RPL already matched the RP one). Presumably it should have brackets to match e.g. the investigating one a few down.

**Review by dracos — CHANGES_REQUESTED** [review]
What you've done is fine, but I think (assume) the existing code had a bug we should probably fix.

File: perllib/Open311/Endpoint/Integration/UK/Bexley/Symology.pm (hunk 1 above)
Comment (davea) [reply perllib/Open311/Endpoint/Integration/UK/Bexley/Symology.pm]: Good spot! Have added brackets.

**Review by dracos — APPROVED** [review]
Woo

**Pull request merged**